src/frontend/src/components/WebformTable/common/types.ts

```diff
@@ -46,7 +46,7 @@ export interface Metadata {
   keepPrivate?: boolean;
   publicId?: string;
   sequencingDate?: string;
-  collectionLocation?: NamedGisaidLocation;
```

Comment:
I just saw the comment on `collectionLocationName` elsewhere
```
  const collectionLocationName = () => {
    // collection location will always be a NamedGisaidLocation at this stage,
    // the only time collectionLocation will be a string is during tsv upload
    // where collectionLocation can be "DELETE" (when a user wants to clear a value)
```
If `string` type is purely used for the DELETE sentinel value, maybe have the other type here not be a generic string, but the constant string of just "DELETE", I think that would be a lot more explicit and clear.

Now, that said, we would need to make sure where the parsing is happening it's only returning either the `NamedGisaidLocation` or the "DELETE" string, so it would require re-writing that part a bit. I think it would make some of the logic around deletion and collectionLocation types less confusing in other places, but I'm not sure it's really worth the effort given time constraints. I dunno.

Reply:
yeah that's a good idea! I will try to change that over in my later PR (since i want this to get merged in soon so maya can start on the review page)